modules/tracker/controllers/components/ApiComponent.php

```diff
@@ -56,9 +56,9 @@ private function _getUser($args)
     }
 
     /**
-     * Associate a result item with a particular scalar value.
+     * Associate a result item with a particular submission.
      *
-     * @param scalarIds Comma separated list of scalar ids to associate the item with
+     * @param submissionId the submission to associate the item with
```

Comment:
is there a reason to take `submissionId` here but `submissionUuid` in `scalarAdd`?